fix: [taskdialog]Every time a small file is copied, the position of the copy window will be refreshed

liyigang1 opened the pull request:
The size of each item will be adjusted when the height of the copy progress bar is changed. Here, the position of the entire display dialog box is adjusted. Modify without adjusting the entire display dialogue position.

Log: Every time a small file is copied, the position of the copy window will be refreshed
Bug: https://pms.uniontech.com/bug-view-199201.html https://pms.uniontech.com/bug-view-199209.html

Commit:
chore: Sync by https://github.com/linuxdeepin/.github/commit/2e5e092ba3f86b16d1aabbabcf0bfd2ae65b19c8

sync by github workflow

Log: none
Influence: none

pppanghu77 committed:
fix: [vault] vault fileview register failed

cant register when plugin initialize()

Log: as des
Bug: https://pms.uniontech.com/bug-view-199339.html

Johnson-zs commented on src/dfm-base/dialogs/taskdialog/taskdialog.cpp:
```diff
@@ -177,7 +184,9 @@ void TaskDialog::adjustSize()
     }
 
     layout()->setSizeConstraint(QLayout::SetNoConstraint);
-    moveYCenter();
```
adjustSize和moveToCenter是两件事情，为何要写到一个接口中呢，外部根据需要显式的直接调用moveToCenter不行吗

liyigang1 replied:
这是以前的代码，没有做过修改。好的立刻修改

Johnson-zs commented on src/dfm-base/dialogs/taskdialog/taskdialog.cpp:
```diff
@@ -150,11 +150,18 @@ void TaskDialog::setTitle(int taskCount)
+void TaskDialog::adjustSize(const bool moveToYcenter)
 {
     if (!adjustSizeMutex)
         adjustSizeMutex = new QMutex();
```
TaskDialog都在主线程中应用，有必要加锁吗？可以直接构造函数断言在主线程中构造

liyigang1 replied:
以前的代码，没有修改。立刻修改

GongHeng2017 committed:
fix: [open-with] the custom desktopfile show

let the custom desktopfile not show

Log: solved problem of open with
Bug: https://pms.uniontech.com/bug-view-199623.html

Kakueeen committed:
fix: Failed to cut files from the trash
When retrieving the original file name of a file from the trash ,the actual information obtained is the file's path

Log: fix bug
Bug: https://pms.uniontech.com/bug-view-199397.html

Kakueeen committed:
chore: update translations
update translations

Log: update translations

Kakueeen committed:
fix: Some HTML files have incorrect icons

Traverse all icon names and determine whether the icons exist in the system theme

Log: fix bug
Bug: https://pms.uniontech.com/bug-view-199403.html

max-lvs reviewed (commented):
提交到 master 分支

liyigang1 committed:
fix: [taskdialog]Every time a small file is copied, the position of the copy window will be refreshed

The size of each item will be adjusted when the height of the copy progress bar is changed. Here, the position of the entire display dialog box is adjusted. Modify without adjusting the entire display dialogue position.

Log: Every time a small file is copied, the position of the copy window will be refreshed
Bug: https://pms.uniontech.com/bug-view-199201.html https://pms.uniontech.com/bug-view-199209.html
